### ISSUE TITLE
Gain agreement on proposed data model changes for Vet Center CAP data

### ISSUE THREAD

**Issue by kmariepat-cityfriends** [department-of-veterans-affairs/va.gov-cms#13613]
#### Background 
Review the following cards for past work done on determining this model:

- Lighthouse discovery: Vet Center CAP data #4410
- CMS - Lighthouse: gain agreement on proposed data model changes #10869

#### Acceptance Criteria 
- [ ] I expect changes are made to the CAP data model to have parody with the model outlined in this card: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4769
- [ ]

**Comment by swirtSJW** [comment]
Delivered to Lighthouse for review here https://dsva.slack.com/archives/C02BTJTDFTN/p1684729527218149

**Comment by swirtSJW** [comment]
This is now waiting on Lighthouse to [review usability of field names](https://dsva.slack.com/archives/C02BTJTDFTN/p1684788189668329?thread_ts=1684729527.218149&cid=C02BTJTDFTN)

**Comment by jilladams** [comment]
Notes from 6/5 lighthouse meeting re: status: 

- Swirt pushed sample data in progress, over the weekend, Adam is taking a look currently 
    - Also looking for better field name,  work with Dana 
- Need lighthouse to determine the end-point approach for update/add/delete of CAPs (Likely Adam)
    - Schema was proposed a while ago, at any point we can act on enacting that schema 
^^ These tasks will satisfy  this ticket.


Still identified as a post-V1 effort. Ticket will roll to Sprint 86.

**Comment by swirtSJW** [comment]
This is the new PR from the branch. https://github.com/department-of-veterans-affairs/va.gov-cms/pull/16837 
It is largely feature complete pending Lighthouse giving us an actual endpoint and coordinating / testing with LH.

**Comment by Agile6MSkinner** [comment]
@davidmpickett @omahane Is this dead now that swirt is gone or should we revive it?

**Comment by davidmpickett** [comment]
I think that this is a still a **when**, not an **if**, question. 

I'm not sure if this is an example of something that is possible on Lighthouse Facilities API v1 that wasn't possible on v0 and just hasn't been prioritized, or if it's blocked until Lighthouse v2. Either way, it's a core piece of the long term strategy for Facilities data. The PR may be stale, but the work is still relevant and has high Veteran value. It would enable surfacing about ~450 facilities in Facility Locator.

The approach here will likely also inform the #15366 as they both involve making Drupal the source of truth for Lighthouse of basic facility data (currently we're only source of truth to them for operating status and service details)

I haven't been super looped in to the Lighthouse convos, others can probably give a more accurate recap of the blockers/ next steps

**Comment by jilladams** [comment]
Card for this in the Lighthouse sync board: https://trello.com/c/ciZEObA6/1-vet-center-community-access-point-push-receive-partially-blocked-on-endpoint.  Blocked on Lighthouse capacity to pick it up / define the endpoint we will push to.
